tutorials/parsl-rp/environment.yml

```diff
@@ -0,0 +1,6 @@
+name: conda-env-rct-py
```

Comment:
this line could be skipped, since we use it to update the existing env `rct`, and `conda-env-rct-py` is for kernel name

Reply:
I think that is what you asked me to do. Did I get it wrong? If it does not create an issue, I would be inclined to ignore.

Reply:
> I think that is what you asked me to do. Did I get it wrong?

about other part, which you updated in this commit (`kernelspec`)

> If it does not create an issue, I would be inclined to ignore.

ok, and confirm that it works just fine (just tested)